BuildingBlocks/src/BuildingBlocks.Application/Attributes/ApplyQuotasForMetricsAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Enmeshed.BuildingBlocks.Domain.StronglyTypedIds;
+
+namespace Enmeshed.BuildingBlocks.Application.Attributes;
+public class ApplyQuotasForMetricsAttribute : Attribute
```

Comment:
Inherit from `ValidationAttribute` instead, override `IsValid` and make sure that the Attribute is applied to a class that inherits from `ICommand`.